README.md

```diff
@@ -1,302 +1,210 @@
 # Cloud SQL Auth proxy
 
 [![CI][ci-badge]][ci-build]
-[![Go Reference][pkg-badge]][pkg-docs]
 
-The [Cloud SQL Auth proxy][proxy-page] is a binary that provides IAM-based
-authorization and encryption when connecting to a Cloud SQL instance.
+The Cloud SQL Auth proxy is the recommended way to connect to your Cloud SQL instance.
+The proxy authorizes all client connections based the IAM principal and secures
+each connection with TLS 1.3 encryption.
 
-See the [Connecting Overview][connection-overview] page for more information on
-connecting to a Cloud SQL instance, or the [About the proxy][about-proxy] page
-for details on how the Cloud SQL proxy works.
+The Cloud SQL Auth proxy has support for:
 
-Note: The Proxy *cannot* provide a network path to a Cloud SQL instance if one
-is not already present (e.g., the proxy cannot access a VPC if it does not
-already have access to it).
+- [Auto IAM Authentication][iam-auth] (Postgres only)
+- [Cloud Monitoring][] and [Cloud Trace][]
+- [Prometheus][]
+- [HTTP Healthchecks][health-check-example] for use with Kubernetes
+- Separate Dialer functionality released as the [Cloud SQL Go Connector][go connector]
+- Fully POSIX-compliant flags
 
-## Installation
-
-For 64-bit Linux, run:
-
-```
-VERSION=v1.21.0 # see Releases for other versions
-wget "https://storage.googleapis.com/cloudsql-proxy/$VERSION/cloud_sql_proxy.linux.amd64" -O cloud_sql_proxy
-chmod +x cloud_sql_proxy
-```
-
-Releases for additional OS's and architectures and be found on the [releases
-page][releases].
+If you're using Go, consider using the [Cloud SQL Go connector][go connector] directly.
```

Comment:
How about the other language connectors?

Reply:
Good idea. Done.